doc/BUILD.bazel

```diff
@@ -0,0 +1,29 @@
+load("@io_bazel_stardoc//stardoc:stardoc.bzl", "stardoc")
+
+stardoc(
```

Comment:
It you put these in the directories with the bzl files, you won't have to export the bzl files.

https://github.com/bazelbuild/stardoc/issues/57 is opened for the rule to maybe better support bzl_library directly for srcs.